autopilot/multiwaypointfollower.cpp

```diff
@@ -142,3 +142,8 @@ void MultiWaypointFollower::receiveDistanceOfRouteLeft(double meters)
 {
     emit distanceOfRouteLeft(meters);
 }
+
+QList<PosPoint> MultiWaypointFollower::getCurrentRoute()
+{
+    return mWaypointList;
```

Comment:
I do not think this is correct, but should be
`return mWaypointFollowerList[mActiveWaypointFollowerID]->getCurrentRoute()`
Please have a chat with @rickaha

Reply:
Solution corrected after discussion with @rickaha